src/config/yargs.ts

```diff
@@ -0,0 +1,61 @@
+// Copyright 2020-2024 SubQuery Pte Ltd authors & contributors
+// SPDX-License-Identifier: GPL-3.0
+
+import {hideBin} from 'yargs/helpers';
+import yargs from 'yargs/yargs';
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export function getYargsOption() {
```

Comment:
Can we unwrap this and use yargs properly please. There should be a runner function. Example here https://github.com/subquery/ai-intents-playground/blob/be5cbf536a3e74751271af83e29ff60b41826a51/src/index.ts#L31

It should greatly simplify how the app starts up